build.gradle

```diff
@@ -55,8 +55,10 @@ compileTestJava {
 
 ext {
 	assertjVersion = '3.4.1'
+	jacksonVersion = '2.6.7'
+	log4jVersion = '2.6.2'
```

Comment:
Not sure what you mean. Sorry.
This one?
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```
All tests pass for me anyway.

I don't mind to go ahead with this change, but I want to understand "why?".
I may need something similar in the other place if I know the reason and cause.

Thanks.

P.S. You ask me to add commit comment to explain changes, but I can't find something on the matter in your commit :wink: